src/components/ModelView/ChartsHolder.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {
+  MainContentWrapper,
+  MainContentInner,
+  MainContentInnerBody,
+} from './ChartsHolder.style';
+import LocationPageHeader from 'components/LocationPageHeader/LocationPageHeader';
+import NoCountyDetail from './NoCountyDetail';
+import ModelChart from 'components/Charts/ModelChart';
+import { Projections } from 'models/Projections';
+import { Projection } from 'models/Projection';
+
+const ChartsHolder = (props: {
+  projections: Projections; // make this be a type with
+  stateId: string;
+  countyId: string;
+}) => {
+  // this is the inferred projection. Ideally we'd pass in just the single projeciton, but the location page header needs the full obj
```

Comment:
technically "primary" might not be inferred if we don't have an inferred one for this county.